lighthouse/datadog_checks/lighthouse/data/conf.yaml.example

```
@@ -8,17 +8,32 @@ instances:
     #
   - min_collection_interval: 60
 
-    ## @param url - string - required
-    ## url of the website to get lighthouse metrics of
+    ## @param url - string - optional
+    ## url of the website to get lighthouse metrics of. This option is deprecated.
     ## ex: "https://www.datadoghq.com"
     #
     url: <URL_PLACEHOLDER>
 
+    ## @param urls - list of strings - required
+    ## urls of the websites to get lighthouse metrics of
+    ## ex: "https://www.datadoghq.com"
+    #
+    urls:
+      - "https://www.datadoghq.com"
+      - "https://docs.datadoghq.com/integrations/lighthouse/"
+      - "https://docs.datadoghq.com/getting_started/"
+    
     ## @param name - string - required
     ## name of your website.
     ## ex: "datadog_homepage"
     #
     name: <NAME_PLACEHOLDER>
+    
+    ## @param form_factor - string - optional
+    ## passed to lighthouse as the form-factor param
+    ## options are "mobile" or "desktop"
+    #
+    form_factor: <FORM_FACTOR_PLACEHOLDER>
```

```suggestion
    # form_factor: <FORM_FACTOR_PLACEHOLDER>
```
Let's comment this out by default since it's optional